models/issue_stress.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
Please set the correct date and change filename to end with _test.go

Reply:
If I used a _test.go ending for the file name, the function was not compiled in the integrations build. Anyway, this PR is not meant to be merged because it always produces a FAIL. 😞